beesdoo_purchase/__manifest__.py

```diff
@@ -5,11 +5,12 @@
         - A filter w.r.t. the mail sellers is placed on the products field of a
         purchase order
         - Allow inverting the Purchase Order Reference on the invoice lines
+        - Allows adapting a product's purchase and/or selling price from a PO.
```

Comment:
This makes the summary entry too long and breaks the module loading on any server that has this module  available. No less :p

So this should be summarized into a one liner since it's redundant w/ DESCRIPTION.rst

Reply:
done :)